docs/source/portal-client.rst

```diff
@@ -0,0 +1,64 @@
+Portal Client
+=============
+
+The ``FractalClient`` is the primary entry point to a ``FractalServer`` instance.
+
+We can initialize a ``FractalClient`` by pointing it to a server instance. If you would like to start your own server see
+the setting up a server (NYI) section.
+
+.. code-block:: python
+
+    >>> import qcfractal.interface as portal
+    >>> client = portal.FractalClient("localhost:8888")
+    >>> client
+    FractalClient(server='http://localhost:8888/', username='None')
+
+The ``FractalClient`` handles all communication to the server from the Python API layer. This core communication access the fundamental structure of the server is often not used in favor of more general handlers. See (Database/TorsionDriveORM/etc).
+
+The ``FractalClient`` can also be initialized from a file which is useful so that addresses and username do not have to be retyped for every line and reduces the chance that a username and password could accidentally be added to a version control system. This can be accessed by creating a new object by ``FractalClient.from_file()`` whereby default the server searches for a ``qcportal_config.yaml`` file in either the current working directory or from the canonical ``~/.qca/`` folder.
```

Comment:
"In ``FractalClient.from_file()``, by default the server searches for a ``qcportal_config.yaml`` file in either the current working directory or from the canonical ``~/.qca/`` folder."